src/Features/Core/Portable/DisposeAnalysis/DisposeAnalysisHelper.cs

```diff
@@ -147,7 +148,8 @@ private void EnsureDisposableFieldsMap()
                         context.Options, rule, _disposeOwnershipTransferLikelyTypes, trackInstanceFields,
                         exceptionPathsAnalysis: false, context.CancellationToken, out pointsToAnalysisResult,
                         interproceduralAnalysisPredicateOpt: interproceduralAnalysisPredicateOpt,
-                        defaultDisposeOwnershipTransferAtConstructor: true);
+                        defaultDisposeOwnershipTransferAtConstructor: true,
+                        defaultDisposeOwnershipTransferAtMethodCall: true);
```

Comment:
I guess this is new flowanalysis stuff you updated version for. and I guess this is for the stack stuff. so, when diposable object is based to other method, we assume by default ownership is transferred?

Reply:
I guess as long as that variable is no longer used after that method call?

Reply:
Yes, the flow analysis utilities now has an editorconfig option to configure if dispose ownersip transfer is assumed for method invocations in metadata. This parameter is the default value if user did not configure it, and we set it to true to avoid false positives in IDE.